storage/badger/operation/prefix.go

```diff
@@ -72,8 +72,8 @@ const (
 	codeEpochSetup         = 61 // EpochSetup service event, keyed by ID
 	codeEpochCommit        = 62 // EpochCommit service event, keyed by ID
 	codeBeaconPrivateKey   = 63 // BeaconPrivateKey, keyed by epoch counter
-	codeDKGStarted         = 64 // flag that the DKG for an epoch has been started
-	codeDKGEnded           = 65 // flag that the DKG for an epoch has ended (stores end state)
+	_                      = 64 // [DEPRECATED] flag that the DKG for an epoch has been started
+	codeDKGState           = 65 // current state of Recoverable Random Beacon State Machine for given epoch
```

Comment:
One possible approach is that we agree that upgrade MUST happen before entering epoch setup phase. If we can upgrade before epoch setup(during the staking phase) we must be safe since there won't be any values in the DB. This will also help with the `flow.DKGIndexMap` since by that point we observe an `flow.EpochCommit` all nodes will be upgraded. Since we can set upgrade by view, we should be able to have a reliable mechanism how to schedule the update.

Reply:
After discussion, created https://github.com/onflow/flow-go/issues/6786 to address this